integration/singularity/store.go

```diff
@@ -431,3 +436,114 @@ func (s *SingularityStore) Describe(ctx context.Context, id blob.ID) (*blob.Desc
 	}
 	return descriptor, nil
 }
+
+func (s *SingularityStore) runCleanupWorker(ctx context.Context) {
+	// Run immediately once before starting ticker
+	go func() {
+		if err := s.cleanup(context.Background()); err != nil {
+			logger.Errorf("Local store cleanup failed: %w", err)
+		}
+	}()
```

Comment:
Refactor the cycle logic into a `func` variable here and call it instead of copying code?